http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1317,11 +1364,12 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
     ev: Ask[F, AppContext]
   ): F[Option[CromwellDatabaseNames]] = {
     val databaseConfigEnabled = app.appType match {
-      case AppType.Cromwell => config.coaAppConfig.databaseEnabled
-      case _                => false
+      case AppType.Cromwell => config.coaAppConfig.databaseEnabled && landingZoneResources.postgresServer.isDefined
+      case AppType.CromwellRunnerApp => true
```

Comment:
I assume it's sorted, but I'm curious how this plays into the WORKFLOWS app and its Cromwell DB

Reply:
Chatted via Slack. For Workflows app [this method](https://github.com/DataBiosphere/leonardo/blob/80f571487b5f904ad109e5f3e890cf1f96716fd6/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala#L1221) is called to create identity and postgres databases